packages/modelserver-node/src/command-provider-registry.spec.ts

```diff
@@ -0,0 +1,79 @@
+/********************************************************************************
+ * Copyright (c) 2022 STMicroelectronics.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
+ * available at https://opensource.org/licenses/MIT.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR MIT
+ *******************************************************************************/
+import { ModelServerCommand, replace } from '@eclipse-emfcloud/modelserver-client';
+import { CommandProvider, Logger, Transaction } from '@eclipse-emfcloud/modelserver-plugin-ext';
+import { expect } from 'chai';
+import { Container } from 'inversify';
+
+import { CommandProviderRegistry } from './command-provider-registry';
+
+describe('CommandProviderRegistry', () => {
+    let registry: CommandProviderRegistry;
+    // let client: ModelServerClientV2;
```

Comment:
Will this `client` make a comeback or can it be deleted?

Reply:
Thanks for catching that, dropped it!